pages/index.tsx

```diff
@@ -1,93 +1,39 @@
-import { useMutation, useQuery } from '@apollo/client'
-import { graphql } from 'lib/gql'
-import Link from 'next/link'
-import { useState } from 'react'
-import { initializeApollo } from '../lib/apollo'
+import Link from "next/link";
+import { useState } from "react";
 
-const updateNameDocument = graphql(/* GraphQL */ `
-  mutation UpdateName($name: String!) {
-    updateName(name: $name) {
-      id
-      name
-      status
-    }
-  }
-`)
+const yogaPosesData = [
+  { src: "#", description: "1. Describing the pose" },
+  { src: "#", description: "2. History of the pose" },
+  { src: "#", description: "3. Context around the pose" },
+];
 
-const viewerDocument = graphql(/* GraphQL */ `
-  query Viewer {
-    viewer {
-      id
-      name
-      status
-    }
-  }
-`)
```

Comment:
What do you think about removing the boilerplate code and clean this up b4 implementing the rest of this?

Reply:
Oops, meant to put this comment on index.tsx, not this file.